zeppelin-interpreter/pom.xml

```diff
@@ -133,6 +133,12 @@
     </dependency>
 
     <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcpkix-jdk15on</artifactId>
+      <version>1.52</version>
```

Comment:
we would need this in the LICENSE file...

Reply:
done